tests/deploy_test.go

```diff
@@ -194,6 +345,25 @@ func Test_Deploy_WithAnnotations(t *testing.T) {
 	}
 }
 
+func scale(t *testing.T, function string, replicas uint64) (int, error) {
+
+	req := map[string]interface{}{"services": function, "replicas": replicas}
```

Comment:
Can you vendor this from the openfaas/faas-provider or openfaas/faas/gateway package?

Reply:
i.e. https://github.com/openfaas/faas-provider/blob/master/types/requests.go#L6